src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx';
+import React, { FC, useState } from 'react';
+import { HexColorPicker } from 'react-colorful';
+
+import './ColorPicker.scss';
+import { ColorPickerProps } from './ColorPicker.types';
+
+const ColorPicker: FC<ColorPickerProps> = ({
+	color,
+	onChange,
+	disabled = false,
+	className,
+	rootClassName: root = 'c-color-picker',
+	style,
+}) => {
+	const [showColorPicker, setShowColorPicker] = useState<boolean>(false);
+
+	return (
+		<div
+			className={clsx(root, className, { 'c-color-picker__disabled': disabled })}
```

Comment:
`c-color-picker` is de rootClassName en zou dus aanpasbaar moeten kunnen zijn. Er is een helper `bemCls` waaraan je deze kan binden. Je vindt hiervan in de andere componenten wel een voorbeeld van hoe dit gebruikt wordt.